### PR TITLE
feat setup skeleton of project

### PR DESCRIPTION
**Pull request by vantam87**
Build common views: 
- Button 
- Textfield
- Label
- Font size
- Font style
- Spacing
- Localization
- Toast message

Screen Shot 2021-10-05 at 09 55 20

### CODE REVIEW

File: Wallet/Resources/Localizable/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* 
+  Localizable.strings
+  Wallet
+
+  Created by Tam Nguyen on 03/10/2021.
+  Copyright © 2021 MVL. All rights reserved.
+*/
+
+"btn_submit" = "Submit";
+"btn_google" = "Continue with Google";
```
Comment (spica) [inline Wallet/Resources/Localizable/en.lproj/Localizable.strings:10]: This is quite ambiguous. 
How about to use 'btn_login_with_google'?
btn_apple is same.
Or 'btn_continue_with_google/apple' seems fine too.

File: Wallet/Sources/Supporting/Helpers/UIKit/FontSize.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+//  FontSize.swift
+//  Wallet
+//
+//  Created by Tam Nguyen on 03/10/2021.
+//  Copyright © 2021 MVL. All rights reserved.
+//
+
+import UIKit
+
+enum FontSize: CGFloat {
```
Comment (spica) [inline Wallet/Sources/Supporting/Helpers/UIKit/FontSize.swift:11]: These font sizes are fixed for our wallet app? Or just an example?

File: Wallet/Resources/Localizable/en.lproj/Localizable.strings (hunk 1 above)
Comment (vantam87) [reply Wallet/Resources/Localizable/en.lproj/Localizable.strings:10]: yup, I'll update it as 'btn_login_with_google' 👍

File: Wallet/Sources/Supporting/Helpers/UIKit/FontSize.swift (hunk 2 above)
Comment (vantam87) [reply Wallet/Sources/Supporting/Helpers/UIKit/FontSize.swift:11]: Yes, these font sizes are fixed for our app. I get it from Figma

Screen Shot 2021-10-05 at 10 24 30

**Review by spica — APPROVED** [review]

**Pull request merged**